examples/experimental/i3s-17-and-debug/components/debug-panel.js

```diff
@@ -87,13 +87,15 @@ export default class DebugPanel extends PureComponent {
       obb: false,
       tileColorMode: INITIAL_TILE_COLOR_MODE,
       obbColorMode: INITIAL_OBB_COLOR_MODE,
-      pickable: false
+      pickable: false,
+      loadTiles: true
     };
 
     this.toggleDebugPanel = this.toggleDebugPanel.bind(this);
     this.toggleMinimap = this.toggleMinimap.bind(this);
     this.toggleObb = this.toggleObb.bind(this);
     this.togglePickable = this.togglePickable.bind(this);
+    this.toggleLoading = this.toggleLoading.bind(this);
```

Comment:
Nit: Normally I use the `_on...` naming convention for callbacks: `_onToggleLoading`.